xarray/coding/cftime_offsets.py

```diff
@@ -209,7 +209,7 @@ def _shift_months(date, months, day_option='start'):
     # the returned date object in versions of cftime between 1.0.2 and
     # 1.0.3.4.  It can be removed for versions of cftime greater than
     # 1.0.3.4.
-    return date.replace(year=year, month=month, day=day, dayofwk=-1)
+    return date.replace(year=year, month=month, day=day)
```

Comment:
Not removing it means that I get a `ValueError: Replacing the dayofyr or dayofwk of a datetime is not supported.` , which is an error message you wrote for cftime. So, I'm a little confused as to what I need to do to here.

Reply:
You must be using the dev version of cftime locally; this will not raise an error in cftime 1.0.3.4.  I agree we'll need to fix this once the new version of cftime is released, but for now let's leave it alone.